src/main/webapp/styles/env.css

```diff
@@ -0,0 +1,110 @@
+.container {
+  width: 60%;
+  margin: 30px auto;
+}
+#open-env {
+  display: none;
+}
+.env {
+  margin: 200px auto 0;
+  background: #e95151;
+  width: 500px;
```

Comment:
Try to use alphabetical order for CSS declarations within a selector. This applies to all selectors.
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order